.github/workflows/check-generate-api.yml

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+---
+
+name: Validate and Generate OpenAPI
+
+on:
+  pull_request:
+    paths:
+      - 'tenancy-api-mapping/apimappingconfigcrs/*.yaml'
```

Comment:
The output is in `tenancy-api-mapping/openapispecs/*.yaml`, so we should check that as well. 

We might check all of `tenancy-api-mapping/` just in case.

Reply:
changed to `tenancy-api-mapping/**`